src/components/DashboardsBar/InformationBlock/LastUpdatedTag.js

```diff
@@ -30,7 +30,7 @@ const LastUpdatedTag = ({ id }) => {
         >
             {(props) => (
                 <div {...props}>
-                    <Tag maxWidth="400px">{message} </Tag>
+                    <Tag maxWidth="200px">{message} </Tag>
```

Comment:
Is this changing the offline updated message? Won't 200px be too short?  
![image](https://github.com/user-attachments/assets/82e1a4f1-95f3-4cc5-a010-76732b63fc41)